### PR TITLE
Update python dependencies

### PR DESCRIPTION
**Pull request by ThiefMaster**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -228,7 +228,7 @@ pynpm==0.1.2
     # via
     #   flask-webpackext
     #   pywebpack
-pyparsing==3.0.1
+pyparsing==2.4.7
```
Comment (pferreir) [inline requirements.txt:231]: Is this intentional? It should be on 3.0.5

Comment (ThiefMaster) [reply requirements.txt:231]: Seems to have been intentional, but apparently got reverted upstream (w/o a new release yet) in the meantime: https://github.com/pypa/packaging/pull/482

Comment (ThiefMaster) [reply requirements.txt:231]: I think it's safe to merge with the downgrade and then upgrade it again once packaging released a version w/o that pin..

Comment (pferreir) [reply requirements.txt:231]: OK, fair enough

**Review by pferreir — APPROVED** [review]

**Pull request merged**